portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Favourite Places</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <a href="index.html" class="button">Home</button>
+    <a href="https://www.linkedin.com/in/yijia-sylvia-zhang" class="button">LinkedIn</a>
+	<a href="https://github.com/yjzhang111" class="button">GitHub</a>
```

Comment:
nit: appears formatting is a little off here, is there a way to have this auto format?